man/journalctl.xml

```diff
@@ -552,12 +553,13 @@
         <term><option>-n</option></term>
         <term><option>--lines=</option></term>
 
-        <listitem><para>Show the most recent journal events and limit the number of events shown. If
-        <option>--follow</option> is used, this option is implied. The argument is a positive integer or
-        <literal>all</literal> to disable line limiting. The default value is 10 if no argument is
-        given.</para>
+        <listitem><para>Show the most recent journal events and limit the number of events shown. The argument
+        is a positive integer or <literal>all</literal> to disable the limit. Additionally, if the number is
+        prefixed with <literal>+</literal>, the oldest journal events are used instead. The default value is
```

Comment:
Well, we document `--reverse` as "Reverse output so that the newest entries are displayed first", and `--lines=` as "Show the most recent journal events and limit the number of events shown". When these are combined, the behavior that corresponds to the documentation is to first order the journal entries from latest to oldest, and then select the latest N entries from them to output.

While the use case of such combination seems narrow, if we change things like that, it's not possible to express the mentioned behavior anymore using purely `journalctl`.

Reply:
And in this PR the combination of `--lines=+N` and `--reverse` is refused, because I reckon it would be even less useful, and it's a new feature that won't break existing semantics.